ca/config.go

```diff
@@ -447,7 +462,11 @@ func RenewTLSConfigNow(ctx context.Context, s *SecurityConfig, connBroker *conne
 	}
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	return s.updateTLSCredentials(tlsKeyPair, issuerInfo)
+	err = s.updateTLSCredentials(tlsKeyPair, issuerInfo)
+	if err != nil && s.queue != nil {
```

Comment:
Can we create a constructor for SecurityConfigs and always ensure there is a queue? (instead of sprinkling checks for nil queues throughout the code?